### PR TITLE
Adds Acast to README.md

### PR DESCRIPTION
**Pull request by JagCesar**

### CODE REVIEW

File: Acast/Fastfile [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    sigh(app_identifier: "com.acast.app.native.production.internalbeta")
+    gym(configuration: "Internal Beta")
+    pilot(app_identifier: "com.acast.app.native.production.internalbeta")
+  end
+
+  desc "Deploy a new version to the App Store"
+  lane :deploy do
+    cocoapods
+    match(app_identifier: "com.acast.app.native.production", type: "appstore")
+    sigh(app_identifier: "com.acast.app.native.production")
```
Comment (KrauseFx) [inline Acast/Fastfile]: Why do you run `sigh` after match? For the development profile?

File: README.md [hunk 2]
```diff
@@ -7,6 +7,33 @@ fastlane Example Setups
 
 This repository contains a few `fastlane` example setups that help you getting started.
 
+### [Acast](https://www.acast.com)
+
+[![Acast logotype](Logos/acast.png)](https://www.acast.com)
+
+🎶 **Simple setup, saving tons of time**
```
Comment (KrauseFx) [inline README.md:14]: :heart:

File: README.md [hunk 3]
```diff
@@ -7,6 +7,33 @@ fastlane Example Setups
...
+
+- Slack notifications on failed builds or unsuccessful tests
+  - Slack notifications are only fired if they're run on our CI
+- Managed provisioning profiles using [Match](https://github.com/fastlane/match)
+- Creation of screenshots using [Snapshot](https://github.com/fastlane/snapshot)
```
Comment (KrauseFx) [inline README.md]: Please change to lowercase `snapshot`

File: README.md [hunk 4]
```diff
@@ -7,6 +7,33 @@ fastlane Example Setups
...
 
+### [Acast](https://www.acast.com)
+
+[![Acast logotype](Logos/acast.png)](https://www.acast.com)
+
+🎶 **Simple setup, saving tons of time**
+
+- Slack notifications on failed builds or unsuccessful tests
+  - Slack notifications are only fired if they're run on our CI
+- Managed provisioning profiles using [Match](https://github.com/fastlane/match)
```
Comment (KrauseFx) [inline README.md]: Same for `match`

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (KrauseFx) [inline .gitignore]: Please add the missing new line

**Comment by KrauseFx** [comment]
This is a _great_ pull request, thank you. I added some smaller comments before it's ready to be merged. Could you also squash your commits to one? Thanks :+1:

File: Acast/Fastfile (hunk 1 above)
Comment (JagCesar) [inline Acast/Fastfile]: Yep. Added Match a few weeks ago and forgot to remove Sigh. Will get rid of this. Thnx! :)

**Comment by JagCesar** [comment]
I've read your comments and updated accordingly. I assumed you wanted frameit, deliver and gym to be spelled with a lowercase as well.

All commits are now squashed to a single one.

Keep up the good work! 🤘🏼

**Comment by KrauseFx** [comment]
This is so great, thanks for the PR and making the changes! :+1: :rocket:

**Comment by KrauseFx** [comment]
Oh I just noticed the `README.md` in the project's directory is missing. Could you run `fastlane docs` to generate it? Do you know that you can use the `desc "[text here]"` method to add documentation to each of the lane?

**Comment by JagCesar** [comment]
Oh sorry, totally forgot about that! I'm on it!

**Pull request merged**